app/src/main/java/ar/com/wolox/android/example/customViews/TermsAndConditions.java

```diff
@@ -0,0 +1,62 @@
+package ar.com.wolox.android.example.customViews;
+
+import android.content.Context;
+import android.content.Intent;
+import android.net.Uri;
+import android.support.annotation.Nullable;
+import android.util.AttributeSet;
+import android.widget.LinearLayout;
+import android.widget.Toast;
+
+import ar.com.wolox.android.R;
+
+/**
+ *
+ */
+public class TermsAndConditions extends LinearLayout {
```

Comment:
There is no need to do this. You can set your string.xml with the text like 
 **<string name="fragment_login_terms_conditions">By joining you agree to our <a href="http://www.wolox.com.ar/">Terms &amp; Conditions</a></string>**

And then in your fragment use **mTermsAndConditions.setMovementMethod(LinkMovementMethod.getInstance());**